frontstage/common/post_event.py

```diff
@@ -27,22 +30,20 @@ def post_event(case_id, description=None, category=None, party_id=None, created_
     :param payload: An optional event payload
     :return: status, message
     """
-    #
     #   Start by making sure we were given a working data set
-    #
     if not (description and category and party_id and created_by):
-        msg = 'description={} category={} party_id={} created_by={}'.format(
-            description, category, party_id, created_by
-        )
-        ons_env.logger.error('Insufficient arguments', arguments=msg)
+        logger.error('Insufficient arguments',
+                     description=description,
+                     category=category,
+                     party_id=party_id,
+                     created_by=created_by)
         return 500, {'code': 500, 'text': 'insufficient arguments'}
```

Comment:
yeah I'm not sure about the post event thing in general, we should probably refactor as part of a later card